source/jormungandr/jormungandr/scenarios/helper_classes/helper_utils.py

```diff
@@ -370,7 +375,7 @@ def _build_crowfly(pt_journey, entry_point, mode, places_free_access, fallback_d
 
     section_datetime = fallback_logic.get_pt_section_datetime(pt_journey)
     pt_datetime = fallback_logic.get_journey_bound_datetime(pt_journey)
-    fallback_duration = fallback_durations[pt_obj.uri].duration
+    fallback_duration = fallback_durations[pt_obj.uri]
```

Comment:
why now this is not the duration field used

Reply:
fallback_durations is now a map of "uri" vs "duration"